src/warnet/admin.py

```diff
@@ -33,3 +36,67 @@ def init():
         f"[green]Copied network and namespace example files to {Path(current_dir) / NETWORK_DIR.name}[/green]"
     )
     richprint(f"[green]Created warnet project structure in {current_dir}[/green]")
+
+
+@click.command()
+@click.argument("kube_config", type=str)
+def auth(kube_config: str) -> None:
```

Comment:
hm, this is not an `admin` command. its meant to be used by warnet users , where an admin has created a cube-config file for them, which they then run `warnet auth <file>`, which will update their `kubectl` config and switch to the appropriate context. I like that you moved this out of `main.py` so I don't think moving it back to main is a good idea but not sure where it should live. Could go into `project.py` or we could create a `users.py` so that it's clear which commands are for the user and which commands are for the admin (by putting them in `admin.py`) ?